app/views/students/index.html.erb

```diff
@@ -1,48 +1,49 @@
-<p id="notice"><%= notice %></p>
+<div class="container">
+  <h1>Course Roster</h1>
 
-<h1>Course Roster</h1>
+  <p>Upload a new roster:</p>
+  <%= form_tag students_import_path, multipart: true, class: "form-inline", style: "display:inline-block;float" do %>
+    <%= file_field_tag :file, class: "form-control" %>
+    <%= submit_tag "Upload", class: "btn btn-primary form-control" %>
+  <% end %>
+  <%= link_to "Download Roster as CSV", students_path(:format => "csv"), method: :get, class: "btn btn-success", style: "display:inline-block" %>
 
-<p>Upload a new roster:</p>
-<%= form_tag students_import_path, multipart: true, class: "form form-inline" do %>
-  <%= file_field_tag :file, class: "form-control" %>
-  <%= submit_tag "Upload", class: "btn btn-primary form-control" %>
-<% end %>
-
-<table class="table">
-  <thead>
-    <tr>
-      <th>Perm</th>
-      <th>First name</th>
-      <th>Last name</th>
-      <th>Email</th>
-      <th>Username</th>
-      <th>Org Membership</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @students.each do |student| %>
+  <table class="table">
+    <thead>
       <tr>
-        <td><%= student.perm %></td>
-        <td><%= student.first_name %></td>
-        <td><%= student.last_name %></td>
-        <td><%= student.email %></td>
-        <% if !student.username.nil? and !student.username.empty? %>
-          <td><a href="https://github.com/<%= student.username %>"><%= student.username %></td>
-          <td><%= is_org_member student.username %></td>
-        <% else %>
-          <td>TBA</td>
-          <td>TBA</td>
-        <% end %>
-        <td><%= link_to 'Show', student %></td>
-        <td><%= link_to 'Edit', edit_student_path(student) %></td>
-        <td><%= link_to 'Destroy', student, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <th>Perm</th>
+        <th>First name</th>
+        <th>Last name</th>
+        <th>Email</th>
+        <th>Username</th>
```

Comment:
Ah, this is the right way to make a line comment. 

"I wonder if THIS is the place to make the change from "username" to "github username"

https://anacapa-grader.slack.com/archives/general/p1483658966000182